Cargo.toml

```diff
@@ -18,7 +18,7 @@ home = "0.5.3"  # use same ver in build-dep
 env_logger = "0.8"
 
 [build-dependencies]
-ureq = "1.0"  # allows to keep MSRV 1.41.1
+ureq = "2.1"
```

Comment:
I hope it's ok to use the same version as electrsd

Reply:
sorry for not giving an update on this but I prefer to put this in standby at the moment